.pfnci/common.sh

```diff
@@ -12,20 +12,27 @@ gcloud auth configure-docker
 TEMP=$(mktemp -d)
 mount -t tmpfs tmpfs ${TEMP}/ -o size=100%
 
+function get_local_version() {
```

Comment:
```suggestion
get_local_version() {
```

Reply:
This succeeds with `bash` but fails with `sh`.

Reply:
Oops, thanks for the catch!